mathesar/models.py

```diff
@@ -156,6 +156,16 @@ def sa_column_names(self):
     def has_dependencies(self):
         return True
 
+    @property
+    def pk_type(self):
```

Comment:
I think instead of this property, it would be cleaner to have a `num_primary_keys` property that just returned the number.